modules/compliance-custom-node-pools.adoc

```diff
@@ -75,14 +54,8 @@ profiles:
 - apiGroup: compliance.openshift.io/v1alpha1
   kind: TailoredProfile
   name: cis-example-tp
-settingsRef:
```

Comment:
The settingsRef should not be deleted. You need to remove the TailoredProfile before this line. 
```
- apiGroup: compliance.openshift.io/v1alpha1
  kind: TailoredProfile
  name: cis-example-tp
```

Reply:
This is done, thank you @xiaojiey !

Reply:
Still not correct. The ScanSettingBinding CR should be as below. Thanks.
```
apiVersion: compliance.openshift.io/v1alpha1
kind: ScanSettingBinding
metadata:
  name: cis
  namespace: openshift-compliance
profiles:
- apiGroup: compliance.openshift.io/v1alpha1
  kind: Profile
  name: ocp4-cis
- apiGroup: compliance.openshift.io/v1alpha1
  kind: Profile
  name: ocp4-cis-node
settingsRef:
  apiGroup: compliance.openshift.io/v1alpha1
  kind: ScanSetting
  name: default
```

Reply:
@xiaojiey Done, thank you!

Reply:
Need to remove
```
- apiGroup: compliance.openshift.io/v1alpha1
  name: cis-example-tp
```

Reply:
@xiaojiey This is done, thank you!

Reply:
@xiaojiey I believe I have the YAML correct now, PTAL.